### PR TITLE
Epmlstrtai-861: Remove email propertysource

### PR DESCRIPTION
**Pull request by Sadredneck**

### CODE REVIEW

File: bdd-generator/src/main/java/com/epam/test_generator/services/EmailServiceImpl.java [hunk 1]
```diff
@@ -71,7 +68,7 @@ public void sendResetPasswordMessage(User user, HttpServletRequest request) {
         String subject = environment.getProperty("subject.password.message");
         String text = environment.getProperty("reset.password.message");
         text = String.format(text, user.getName(), user.getSurname(), resetUrl,
-            javaMailSender.getUsername());
+            "TODO");
```
Comment (hdghg) [inline bdd-generator/src/main/java/com/epam/test_generator/services/EmailServiceImpl.java]: Resolve the TODO

File: bdd-generator/src/main/java/com/epam/test_generator/config/WebConfig.java [hunk 2]
```diff
@@ -17,21 +12,19 @@
...
-     * Specialization of PlaceholderConfigurerSupport that resolves ${...} placeholders
-     * within bean definition property values and @Value annotations against the
-     * current Spring Environment and its set of PropertySources.
+     * Specialization of PlaceholderConfigurerSupport that resolves ${...} placeholders within bean
+     * definition property values and @Value annotations against the current Spring Environment and
+     * its set of PropertySources.
+     *
      * @return property sources placeholder configurer bean
      */
     @Bean
```
Comment (hdghg) [inline bdd-generator/src/main/java/com/epam/test_generator/config/WebConfig.java]: I doubt if this bean still required, please investigate and answer here why it is needed/remove it

File: bdd-generator/src/main/resources/application.yml [hunk 3]
```diff
@@ -3,4 +3,41 @@ spring.datasource.url: jdbc:h2:file:~/h2/app_db
...
+    password: cucumberadmin
+    properties:
+      mail:
+        smtp:
+          auth: true
+          starttls:
+            enable: true
+    protocol: smtp
+subject.registration.message: Email verification
+site.name: "https://www.bddgenerator.com"
```
Comment (hdghg) [inline bdd-generator/src/main/resources/application.yml]: double quotes are not required

File: bdd-generator/src/test/resources/application.yml [hunk 4]
```diff
@@ -3,4 +3,41 @@ spring.datasource.url: jdbc:h2:mem:testDB;DB_CLOSE_DELAY=-1
...
+    username: cucumbervarificator@gmail.com
+    password: cucumberadmin
+    properties:
+      mail:
+        smtp:
+          auth: true
+          starttls:
+            enable: true
+    protocol: smtp
+subject.registration.message: Email verification
```
Comment (hdghg) [inline bdd-generator/src/test/resources/application.yml]: Duplication is not required here. Tests would pass if you leave only:

```
spring.datasource.driver-class-name: org.h2.Driver
spring.datasource.url: jdbc:h2:mem:testDB;DB_CLOSE_DELAY=-1
spring.datasource.username: sa
spring.datasource.password: sa
spring.jpa.hibernate.naming.physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
spring.liquibase.change-log: classpath:db/changelog/db.changelog-master.xml
spring:
  mail:
    default-encoding: UTF-8
    host: smtp.gmail.com
    port: 587
    username: test
    password: test
subject.registration.message: Email verification
site.name: https://www.bddgenerator.com
registration.message: registration message
subject.password.message: password message
reset.password.message: password reset message
```

File: bdd-generator/src/main/resources/application.yml (hunk 3 above)
Comment (Sadredneck) [reply bdd-generator/src/main/resources/application.yml]: Removed

File: bdd-generator/src/test/resources/application.yml (hunk 4 above)
Comment (Sadredneck) [reply bdd-generator/src/test/resources/application.yml]: Removed duplicates

**Pull request merged**